chronograf.go

```diff
@@ -380,7 +380,7 @@ type Dashboard struct {
 
 // DashboardCell holds visual and query information for a cell
 type DashboardCell struct {
-	ID      string           `json:"-"`
+	ID      string           `json:"i"`
```

Comment:
The cell id is _now being_ used! I wanted to expose it as `i` in the cell json response. Is this the right thing to do? I just tried it out and it worked.